src/search/containers/search-form/search-form.component.ts

```diff
@@ -131,6 +131,48 @@ export class SearchFormComponent implements OnInit, OnDestroy {
 
     // Set the form control validators
     this.setValidators();
+
+    // Pre-populate the form with existing search parameters
+    const searchParameters: SearchParameters = this.searchService.retrieveState(SearchStatePersistenceKey.SEARCH_PARAMS);
+    if (searchParameters) {
+      // Note: Intentional use of != throughout this if block, to check for search parameter values being not null and not undefined
+      /* eslint-disable eqeqeq */
+      /* tslint:disable:triple-equals */
+      const caseReferences = searchParameters.caseReferences;
+      if (caseReferences != null) {
+        this.formGroup.get(SearchFormControl.CASE_REF).setValue(caseReferences[0] != null ? caseReferences[0] : '');
+      }
+      const otherReferences = searchParameters.otherReferences;
+      if (otherReferences != null) {
+        this.formGroup.get(SearchFormControl.OTHER_REF).setValue(otherReferences[0] != null ? otherReferences[0] : '');
+      }
+      this.formGroup.get(SearchFormControl.FULL_NAME).setValue(searchParameters.fullName != null ? searchParameters.fullName : '');
```

Comment:
Instead of checking the null could you use .setValue(searchParameters.fullName || '')

Reply:
@paulhowes-HMCTS - ah, that is a good point... I like it! I'll change it for the others, too.